charts/device-plugin-operator/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: intel-device-plugins-operator
+description: A Helm chart for Intel Device Plugins Operator for Kubernetes
+type: application
+version: 0.1.0
```

Comment:
if we wish to release an updated chart for a given device pluginsrelease, perhaps the chart `version` should follow the same 0.24.0 etc. versioning.

Reply:
With Helm, there seems to be two ways to handle the `version` and `appVersion`.

1. both are the same
2. `version` is the helm chart version and `appVersion` refers to the application version, in this case to the device-plugin operator version. 

Option 2 allows updates to Helm chart independently of the application. E.g., we can add more parameters to the Helm, bump up the `version` but still use the same `appVersion`.

I am fine with both but prefer option 2.

Reply:
`version` should somehow indicate which device plugin release it's based on because the templates (and crds) content always comes from that release. `AppVersion` is more like the image version and can also be used with the image tag: `{{ .Values.image.tag | default .Chart.AppVersion }}`.

Reply:
As I said, I am fine either way.

@obaranov1 is you change both to 0.24.0 we can get moving.

This is what Helm docs say:

```console
# This is the chart version. This version number should be incremented each time you make changes
# to the chart and its templates, including the app version.
# Versions are expected to follow Semantic Versioning (https://semver.org/)
version: 0.1.0

# This is the version number of the application being deployed. This version number should be
# incremented each time you make changes to the application. Versions are not expected to
# follow Semantic Versioning. They should reflect the version the application is using.
# It is recommended to use it with quotes.
appVersion: "1.16.0"
```

Reply:
done